tac/project-updates/2025/2025-Q2-Hyperledger-Identus.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: 2025 Q2 Hyperledger Identus
+parent: 2025
+grand_parent: Project Updates
+---
+
+# Project Health
+
+The [LF Decentralized Trust Identus LFX Insight Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview/github?project=identus&routedFrom=Github&bestPractice=false&repository=all&dateFilters=Last%20Quarter&dateRange=2025-01-01%20to%202025-03-31&compare=PP&granularity=week&hideBots=true) indicates growth in both contributors and commits to the Identus Platform compared to the previous quarter.
+
+While the primary maintainer of the Identus Platform is the Input Output (IOG) organization, community members and external organizations have also actively contributed, including:
+- [Blocktrust](https://www.blocktrust.dev/)
+- [Socious.io](https://socious.io/)
+- [Ahau.io](https://ahau.io/)
+
+---
+
+# Questions/Issues for the TAC
+
+The Identus team is currently revising its roadmap and removing items that have lost priority or relevance. As part of this effort, the following capability is being put on hold:
+
+- **UAS wrapper** – This feature is considered more of an R&D capability, and there is currently no real customer demand for it.
+
+---
+
+# Releases
+
+The migration of Identus Platform repositories under the `hyperledger-identus` GitHub organization has temporarily slowed the release cadence.
+
+The team is now focused on the upcoming [Identus Platform v2.15 release](https://github.com/hyperledger-identus/.github/releases/tag/v2.15), expected by the end of April.
+
+---
+
+# Overall Activity in the Past Quarter
+
+The Identus community remains active across two Discord servers:
+- **LF Decentralized Trust (LFDT)**
+- [**Identus Community Discord**](https://discord.gg/SCB2eqU4)
+
+Community members continue to ask questions, report issues, and engage in discussions about various use cases. Both the core team and community contributors are actively supporting these interactions.
+
+---
+
+# Current Plans
+
+The strategic direction remains consistent with the annual report. The main goals are to:
+
+- Reduce the maintenance cost and complexity of the Identus Platform
+- Increase adoption among developers and organizations
+- Enhance interoperability with other Self-Sovereign Identity (SSI) platforms
+- Improve documentation and developer tutorials
+
+Planned features for the upcoming year include:
+- Cardano Identus VDR
+- Issuance capabilities in the SDKs
+- OIDC support: OID4VCI, OID4VP, and SIOP 2.0
+- Authentication via the PRISM DID method
+
+Additional features under consideration:
+- Support for JSON-LD Verifiable Credentials
+- Support for OpenBadges 3.0 Verifiable Credentials
+- Integration with WebWallets
+
+To align with Hyperledger Project Lifecycle Criteria, the team also plans to:
+- Enable the OpenSSF Best Practices Badge
+- Enable the OpenSSF Scorecard Badge
```

Comment:
Thanks. Please try to enable these sooner rather than later. The Scorecard especially is not just a way for the TAC to monitor the health of a project; it also gives maintainers insights into their code base that may otherwise not be apparent, and allows them to fix gaps and vulnerabilities.